CHANGELOG.md

```diff
@@ -75,6 +73,10 @@
 * [BUGFIX] Query-frontend: fix wrong query sharding results for queries with boolean result like `1 < bool 0`. #2558
 * [BUGFIX] Fixed error messages related to per-instance limits incorrectly reporting they can be set on a per-tenant basis. #2610
 * [BUGFIX] Perform HA-deduplication before forwarding samples according to forwarding rules in the distributor. #2603
+* [BUGFIX] Fix reporting of tracing spans from PromQL engine. #2707
+* [BUGFIX] Distributor: Apply distributor instance limits before running HA deduplication. #2709
```

Comment:
If this is part of 2.2, then it doesn't need its own changelog entry. We can just mention PR number next to #2603.